src/stories/components/app-bar/notifications/notifications.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from 'lit';
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { customElementStoryRenderer, generateCustomElementArgTypes, standaloneStoryParams } from '../../../utils';
+
+import '@tylertech/forge/app-bar';
+import { IMenuOption } from '@tylertech/forge';
+
+const component = 'forge-app-bar-notification-button';
+
+const meta = {
+  title: 'Components/App Bar/Notifications',
+  render: args => customElementStoryRenderer(component, args),
+  component,
+  parameters: {
+    actions: { disable: true }
+  },
+  argTypes: {
+    ...generateCustomElementArgTypes({
+      tagName: component
+    })
+  },
+  args: {
+  },
+} satisfies Meta;
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Demo: Story = {
+  ...standaloneStoryParams,
+  render: () => {
+    return html`
+    <forge-app-bar title-text="Menu Button">
+      <forge-app-bar-notification-button slot="end" show-badge="true" dot="true">
```

Comment:
This is minor, but whenever we have boolean attributes can we leave off the `="true"` or `="false"`? The existence of the attribute on the element sets the underlying state to `true` and otherwise `false`.